### ISSUE TITLE
Leaderboard analytics 

### ISSUE THREAD

**Issue by phelios** [emojirades/emojirades#187]
I think there are a lot of fun insights that could be extracted from the history.

- Line chart of accumulated points per player over time.
- Bar charts of points per day/week/month

I guess we can start with having the data easier to access and could be as simple as using a Tableau Public or other free visualisation tools.

**Comment by michael-robbins** [comment]
Given we're in the AWS ecosystem, the JSON file lives in S3, you might be able to natively integrate with that?

Only thing is sensitive values stored in the file(s)?

**Comment by phelios** [comment]
yeap, Tableau should be able to connect to public S3 files.

Wait... thought this lives in your beer-fridge?

Sensitive values like the rade? or like PII?

**Comment by michael-robbins** [comment]
Yeah if the 'rade is, or peoples names/etc, hell even Slack ID's I'd prefer not to make public. TBH if we could give some read only creds for the S3 JSON to tableau just so the file isn't public

**Comment by michael-robbins** [comment]
The bot runs on a server at my place, but the durability comes from the scores/gamestate living in S3

**Comment by phelios** [comment]
hehe, just thought of another wild idea, could use one of those fancy python library (matplotlib?) to plot an image and post it as slack message?

**Comment by michael-robbins** [comment]
tbh yes, you absolutely could!

Generate the image in memory then: https://slack.dev/python-slackclient/basic_usage.html#uploading-files

Probably using 'content' attr for the image itself.

**Comment by michael-robbins** [comment]
@Manioc909 if you at some point play with python graphing libraries, would love to have a go at building something like this. With the bot building the scoreboard live then uploading the chart to the channel.

Keeping it all private to a workspace would be my preferred option!

**Comment by michael-robbins** [comment]
We could then build out the command to be more and more complex, like charting individual users, comparing two users over a given timeframe, etc
